env/numerical_compliance.asciidoc

```diff
@@ -304,10 +304,10 @@ devices given as ULP values.
 
 // n + n-1  Number of operations from n multiples and (n-1) additions
 // 2n - 1
-| *OpExtInst* *dot*
-| absolute error tolerance of 'max * max * (2n - 1) * HALF_EPSILON', for vector width _n_ and maximum input operand magnitude _max_ across all vector components
+| *OpDot*
```

Comment:
The other instructions seem to be ordered with core instructions before extended instructions, so maybe move this to after OpFDiv?

Reply:
I went back and forth several times and ultimately decided to keep it in the same place.  This roughly means it matches the tables in the OpenCL C spec, and rightly keeps the instructions alphabetized.  I'm flexible though - if more people think it makes sense to list the core instructions first I don't mind moving it.